packages/ui-components/web/src/Components/FocusManager.tsx

```diff
@@ -5,6 +5,7 @@ import type { Options as FocusTrapOptions } from 'focus-trap';
 interface FocusManagerProps {
   focusTrapOptions?: FocusTrapOptions;
   isMobile?: boolean;
+  onKeyDown?: (e) => void;
```

Comment:
```suggestion
  onKeyDown?: KeyboardEventHandler;
```

import type { KeyboardEventHandler } from 'react';